apps/charterafrica/src/payload/utils/mapLinkAndNameforContributorsInToolsCollection.js

```diff
@@ -0,0 +1,29 @@
+import getPageUrlUsingPayload from "./getPageUrlUsingPayload";
+
+const mapLinkAndNameforContributorsInToolsCollection = async ({
```

Comment:
Why do we need to do this? Doesn't having the hook execute on contributors will automatically appear anywhere contributors are pulled? Do we need really to write something specific for contributors in tools?